### PR TITLE
support htc

### PR DESCRIPTION
**Pull request by irexyc**
## Motivation

address https://github.com/open-mmlab/mmdeploy/issues/2409

### CODE REVIEW

File: docs/en/04-supported-codebases/mmdet.md [hunk 1]
```diff
@@ -214,6 +214,7 @@ Besides python API, mmdeploy SDK also provides other FFI (Foreign Function Inter
 |                 [CenterNet](https://github.com/open-mmlab/mmdetection/tree/main/configs/centernet)                  |   Object Detection    |      Y      |    Y     |  N   |   ?   |    Y     |
 |                    [RTMDet](https://github.com/open-mmlab/mmdetection/tree/main/configs/rtmdet)                     |   Object Detection    |      Y      |    Y     |  N   |   ?   |    Y     |
 |           [Cascade Mask R-CNN](https://github.com/open-mmlab/mmdetection/tree/main/configs/cascade_rcnn)            | Instance Segmentation |      Y      |    Y     |  N   |   N   |    Y     |
+|                       [HTC](https://github.com/open-mmlab/mmdetection/tree/main/configs/htc)                        | Instance Segmentation |      Y      |    Y     |  N   |   ?   |    Y     |
```
Comment (RunningLeon) [inline docs/en/04-supported-codebases/mmdet.md:217]: visualization fails with trt
```
[09/20/2023-02:10:51] [TRT] [E] 1: [cudaDriverHelpers.cpp::operator()::94] Error Code 1: Cuda Driver (an illegal memory access was encountered)
Traceback (most recent call last):
  File "/root/workspace/mmdeploy/mmdeploy/utils/utils.py", line 41, in target_wrapper
    result = target(*args, **kwargs)
  File "/root/workspace/mmdeploy/mmdeploy/apis/visualize.py", line 72, in visualize_model
    result = model.test_step(model_inputs)[0]
  File "/usr/local/lib/python3.8/dist-packages/mmengine/model/base_model/base_model.py", line 145, in test_step
    return self._run_forward(data, mode='predict')  # type: ignore
  File "/usr/local/lib/python3.8/dist-packages/mmengine/model/base_model/base_model.py", line 340, in _run_forward
    results = self(**data, mode=mode)
  File "/usr/local/lib/python3.8/dist-packages/torch/nn/modules/module.py", line 1501, in _call_impl
    return forward_call(*args, **kwargs)
  File "/root/workspace/mmdeploy/mmdeploy/codebase/mmdet/deploy/object_detection_model.py", line 299, in forward
    self.postprocessing_results(batch_dets, batch_labels, batch_masks,
  File "/root/workspace/mmdeploy/mmdeploy/codebase/mmdet/deploy/object_detection_model.py", line 189, in postprocessing_results
    outputs = End2EndModel.__clear_outputs(tmp_outputs)
  File "/root/workspace/mmdeploy/mmdeploy/codebase/mmdet/deploy/object_detection_model.py", line 100, in __clear_outputs
    inds = test_outputs[0][i, :, 4] > 0.0
RuntimeError: CUDA error: an illegal memory access was encountered
CUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.
For debugging consider passing CUDA_LAUNCH_BLOCKING=1.
Compile with `TORCH_USE_CUDA_DSA` to enable device-side assertions.

09/20 02:10:52 - mmengine - ERROR - ./tools/deploy.py - create_process - 82 - visualize tensorrt model failed.
```

Comment (irexyc) [reply docs/en/04-supported-codebases/mmdet.md:217]: Can you share the used command?

**Comment by RunningLeon** [comment]
@irexyc hi，could fix conflicts.

**Review by RunningLeon — APPROVED** [review]
LGTM

**Review by grimoire — APPROVED** [review]
LGTM

**Pull request merged**